src/pages/editor/ChannelClaimFilter.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, FormControl, InputLabel, MenuItem, Select } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useSearchParams } from 'react-router';
+import { ChannelClaimParams } from '../../api/searchApi';
+import { dataTestId } from '../../utils/dataTestIds';
+
+const selectLabelId = 'select-label-id';
+
+export const ChannelClaimFilter = () => {
+  const { t } = useTranslation();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  return (
+    <Box sx={{ display: 'flex', gap: '1rem', my: '1rem', justifyContent: 'end' }}>
+      <FormControl>
+        <InputLabel id={selectLabelId}>{t('tasks.display_options')}</InputLabel>
+        <Select
+          data-testid={dataTestId.editor.channelClaimFilterSelect}
+          sx={{ bgcolor: 'white', minWidth: '12rem' }}
+          labelId={selectLabelId}
+          label={t('tasks.display_options')}
```

Comment:
Tidligere har vi ofte brukt `<TextField select>` i stedet for `<Select>` da den hånterer labels bedre. Noe rart med at man må oppgi både `label` som prop og bruke `<InputLabel>` med id bare for å få inne en label. I tilfelle her får vi da tatt vekk både `FromControl`, `InputLabel` og `labelId`

```suggestion
      <TextField
        select
        data-testid={dataTestId.editor.channelClaimFilterSelect}
        sx={{ bgcolor: 'white', minWidth: '12rem' }}
        label={t('tasks.display_options')}
```